libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -34,6 +34,16 @@ export class BucketClient implements IBucketClient {
     );
   }
 
+  /**
+   * Check if an object exists in the bucket
+   *
+   * @param key Key of the object
+   */
+  public async exists(key: string): Promise<boolean> {
+    const blobClient = this.containerClient.getBlobClient(key);
+    return blobClient.exists();
+  }
```

Comment:
Can you please add a unit test? 
(I can test it on Azure before the PR will be merged)